backend/src/openbeheer/informatieobjecttypen/api/views.py

```diff
@@ -153,10 +163,48 @@ class InformatieObjectTypeDetailView(
     return_data_type = DetailResponseWithoutVersions[InformatieObjectType]
     endpoint_path = "informatieobjecttypen/{uuid}"
 
-    # TODO: not sure if we should expand the zaaktypen, since the IOT will be
-    # shown under a specific zaaktype. But maybe all the other zaaktypen should be
-    # expanded?
     expansions = {}
 
     def get_fieldsets(self) -> FrontendFieldsets:
         return INFORMATIEOBJECTTYPE_FIELDSETS
+
+    def get_fields(
+        self,
+        data: InformatieObjectType,
+        option_overrides: Mapping[str, list[OBOption]] = {},
+        *,
+        base_editable: Callable[[str], bool] = bool,
+    ) -> Iterable[OBField]:
+        # We don't want to edit concept directly, because we use the "publish" action to change it.
+        yield from super().get_fields(
+            data, option_overrides, base_editable=lambda name: name != "concept"
+        )
+
+
+class InformatieObjectTypePublishView(MsgspecAPIView):
+    endpoint_path = "informatieobjecttypen/{uuid}/publish"
+
+    @extend_schema(
+        operation_id="informatieobjecttype_publish",
+        summary="Publish an informatieobjecttype",
+        tags=["Informatieobjecttypen"],
+        responses={
+            204: None,
+            400: ZGWError,
+        },
+    )
+    def post(self, request: Request, slug: str = "", uuid: str = "") -> Response:
```

Comment:
```suggestion
    def post(self, request: Request, slug: str, uuid: UUID) -> Response:
```

I see this is wrong in the ZaaTypePublishView too; feel free to fix :)